### DISCUSSION TITLE
How to change the navigation arrows color in swiper carousel library.

### DISCUSSION THREAD

**Discussion by AguChaves92** [emetdas/Youtube-code#2] [Q&A] [answered]
I have this code in my css. The background of the arrow changes accordingly but the color does not. 

`.swiper-button-prev,
.swiper-button-next {
  background-color: transparent; /* Change to the desired color */
  color: #ffffff; /* Change to the desired color */
}`

Pls Help

**Comment by emetdas** [comment, marked as answer]
.swiper-button-prev ion-icon ,
.swiper-button-nex ion-icon{
----CSS---
}
